changelogs/fragments/70762-sanitize-uri-keys.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
@Shrews This should probably use `security_fixes:`. I already merged #70821 which should use that too. Could you fix this one, and do a new PR to fix the one from #70821?

Reply:
All done! PRs to fix the already merged backports are: PR #71127 for 2.10 and PR #71128 for 2.9 (and already merged the fix to devel in PR #71124).

Reply:
Thanks!